mcs/restapi/admin_groups.go

```diff
@@ -72,17 +72,12 @@ func registerGroupsHandlers(api *operations.McsAPI) {
 }
 
 // listGroups calls MinIO server to list all groups names present on the server.
-func listGroups(client MinioAdmin) (groups []string, err error) {
+func listGroups(client MinioAdmin) ([]string, error) {
```

Comment:
same here, it can be possible to return `[]*string` that allow you to return `nil` instead of  an empty `[]string{}` when there was an error `err`